include/camera/camera.h

```diff
@@ -119,19 +130,19 @@ struct CAMSurface
    void *surfaceBuffer;
    int height; // surface height
    int width; // surface width
-   int unk_0x10; // pitch related?
+   int pitch;
    int alignment; // surface alignment
-   int unk_0x18; // surface tile mode related?
-   int unk_0x1C;
+   int tileMode;
+   int pixelFormat;
```

Comment:
Just curious, where do these fields come from? The `camera.rpl` seems to ignore them completely.

Reply:
Iirc I had these values for a while but I found them in Wii Fit U. I did most of my RE using that.

Reply:
Ah alright.